contracts/ROSCA.sol

```diff
@@ -502,7 +502,7 @@ contract ROSCA {
    * @return int256
    */
   function getParticipantBalance(address user) onlyFromMember external constant returns(int256) {
-    int256 totalCredit = int256(members[user].credit + totalDiscounts);
+    int256 totalCredit = int256(userTotalCredit(user));
```

Comment:
no need to cast to int256

Reply:
you have to cast it, `userTotalCredit` returns a uint256, solidity doesn't allow implicit cast. if you don't have this cast, you get this error 
![image](https://user-images.githubusercontent.com/5243508/34660407-0314b658-f3f7-11e7-9d10-2bcbae3b5070.png)

Reply:
why do we need to use both int256 and uint256?

Reply:
Because the function will return negative value if user    owe the contract by the amount they owe. This is for read only.